configure.ac

```diff
@@ -438,7 +438,7 @@ if test x$enable_jxl = xyes; then
                 jxl_lib=[`find_lib libjxl.dylib`]
                 ;;
             *-*-cygwin* | *-*-mingw*)
-                jxl_lib=[`find_lib "libjxl*.dll"`]
+                jxl_lib=[`find_lib "libjxl.dll"`]
```

Comment:
I settled for the simple `libjxl.dll` since `libjxl(-[0-9]+)?.dll` does not work.

Reply:
What does the cmake'ry look for in mingw cases?

Reply:
It first does a normal `find_library` to find the import library, which is also used for non-shared 3rd party libraries.
Then it uses [`dlltool --identify`](https://github.com/libsdl-org/SDL_image/blob/77a9125036a6f1ed272588cd8b1607e9b37552ca/cmake/PrivateSdlFunctions.cmake#L42) to identify the accompanying dll

Reply:
That's good actually. Can it not be used in autotools side, I wonder.

Reply:
I don't see why not.

CMake knows the path of the import library.
I think autotools will need extra logic to extract this from the output strings of  the `PKG_CHECK_MODULES ` macro.

Reply:
BTW, you even do a similar thing for Mac by picking and poking the second lines below:
```sh
$ PATH=/opt/cross_osx-x86/bin:$PATH i686-apple-darwin9-otool -L lib/libxmp.dylib|grep libxmp
lib/libxmp.dylib:
	/usr/local/lib/libxmp.4.dylib (compatibility version 4.0.0, current version 4.6.0)
$ PATH=/opt/cross_osx-x86/bin:$PATH i686-apple-darwin9-otool -L lib/libxmp.4.dylib|grep libxmp
lib/libxmp.4.dylib:
	/usr/local/lib/libxmp.4.dylib (compatibility version 4.0.0, current version 4.6.0)
```